config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml

```diff
@@ -0,0 +1,88 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-allinone-job
+      decorate: true
+      run_if_changed: 'images/all-in-one'
+      branches:
+        - master
+      spec:
+        serviceAccountName: build-img
+        containers:
+          - image: debian:buster
+            env:
+              - name: DOCKER_CONFIG
+                valueFrom:
+                  secretKeyRef:
+                    name: quay-powercloud-regcred
+                    key: .dockerconfigjson
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                apt-get -y update
+                apt-get install -y curl &&  apt-get install -y wget &&  apt-get install -y golang-go
+                export PATH=$PATH:$GOPATH/bin
+                curl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/`go env GOARCH`/kubectl" -o /usr/local/bin/kubectl
+                chmod +x /usr/local/bin/kubectl
+                wget https://github.com/vmware-tanzu/buildkit-cli-for-kubectl/releases/download/v0.1.2/linux-v0.1.2.tgz
```

Comment:
we cloned using cloneref in other jobs - https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml#L33

Reply:
if we remove this wget then we don't need to install wget command

Reply:
@mkumatag if we use that approach, we have to install make and git. Also, I was getting an error
 ``` GOOS=linux go build -ldflags "-X github.com/vmware-tanzu/buildkit-cli-for-kubectl/version.Version=v0.1.2-0-g25b70ef" -mod=vendor -o bin/linux/kubectl-buildkit ./cmd/kubectl-buildkit
build flag -mod=vendor only valid when using modules
make: *** [Makefile:45: bin/linux/kubectl-buildkit] Error 1
```